packages/cookie-theme-toggler/src/styles.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components'
+import { Button } from '@lidofinance/button'
+
+export const CookieThemeTogglerStyle = styled((props) => (
+  <Button {...props} />
+)).attrs({
```

Comment:
I'm unsure but I remember you could just write `styled(Button)` for this case

Reply:
Yes. But I dont know someone will be use props like size btn and other (my suggest is no - so I will remove props)